flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/Dispatcher.java

```diff
@@ -861,14 +861,18 @@ public void grantLeadership(final UUID newLeaderSessionID) {
 			getMainThreadExecutor());
 	}
 
-	protected CompletableFuture<Void> getJobTerminationFuture(JobID jobId) {
+	CompletableFuture<Void> getJobTerminationFuture(JobID jobId) {
```

Comment:
Should be `private`.

Reply:
It cannot be private since the `TestingDispatcher` needs to acces it.